amundsenatlastypes/schema/01_4_table_report.json

```diff
@@ -0,0 +1,57 @@
+{
+  "entityDefs": [
+    {
+      "superTypes": [
+        "DataSet"
+      ],
+      "name": "TableReport",
+      "description": "External data reports ( url's) associated with table",
+      "typeVersion": "2.0",
+      "attributeDefs": [
+				{
+          "name": "name",
+          "typeName": "string",
+          "isOptional": false,
+          "cardinality": "SINGLE",
+          "valuesMinCount": 1,
+          "valuesMaxCount": 1,
+          "isUnique": false,
+          "isIndexable": true
+        },
+        {
+          "name": "url",
+          "typeName": "string",
+          "isOptional": false,
+          "cardinality": "SINGLE",
+          "valuesMinCount": 1,
+          "valuesMaxCount": 1,
+          "isUnique": false,
+          "isIndexable": true
+        }
+      ]
+    }
+  ],
+  "relationshipDefs": [
+    {
+      "name": "Table__TableReport",
```

Comment:
the relationship should be more generic - between `Report` and `Referenceable`. This way report can be connected to any type of entity.

Reply:
changed in 030fe2b